arbnode/delayed_seq_reorg_test.go

```diff
@@ -1,153 +0,0 @@
-	tracker, err := NewInboxTracker(db, streamer, nil, DefaultSnapSyncConfig)
-	Require(t, err)
-
-	err = streamer.Start(ctx)
-	Require(t, err)
-	exec.Start(ctx)
-	init, err := streamer.GetMessage(0)
-	Require(t, err)
-
-	initMsgDelayed := &DelayedInboxMessage{
-		BlockHash:      [32]byte{},
-		BeforeInboxAcc: [32]byte{},
-		Message:        init.Message,
-	}
-	delayedRequestId := common.BigToHash(common.Big1)
-	userDelayed := &DelayedInboxMessage{
-		BlockHash:      [32]byte{},
-		BeforeInboxAcc: initMsgDelayed.AfterInboxAcc(),
-		Message: &arbostypes.L1IncomingMessage{
-			Header: &arbostypes.L1IncomingMessageHeader{
-				Kind:        arbostypes.L1MessageType_EndOfBlock,
-				Poster:      [20]byte{},
-				BlockNumber: 0,
-				Timestamp:   0,
-				RequestId:   &delayedRequestId,
-				L1BaseFee:   common.Big0,
-			},
-		},
-	}
-	err = tracker.AddDelayedMessages([]*DelayedInboxMessage{initMsgDelayed, userDelayed}, false)
-	Require(t, err)
-
-	serializedInitMsgBatch := make([]byte, 40)
-	binary.BigEndian.PutUint64(serializedInitMsgBatch[32:], 1)
-	initMsgBatch := &SequencerInboxBatch{
-		BlockHash:              [32]byte{},
-		ParentChainBlockNumber: 0,
-		SequenceNumber:         0,
-		BeforeInboxAcc:         [32]byte{},
-		AfterInboxAcc:          [32]byte{1},
-		AfterDelayedAcc:        initMsgDelayed.AfterInboxAcc(),
-		AfterDelayedCount:      1,
-		TimeBounds:             bridgegen.IBridgeTimeBounds{},
-		rawLog:                 types.Log{},
-		dataLocation:           0,
-		bridgeAddress:          [20]byte{},
-		serialized:             serializedInitMsgBatch,
-	}
-	serializedUserMsgBatch := make([]byte, 40)
-	binary.BigEndian.PutUint64(serializedUserMsgBatch[32:], 2)
-	userMsgBatch := &SequencerInboxBatch{
-		BlockHash:              [32]byte{},
-		ParentChainBlockNumber: 0,
-		SequenceNumber:         1,
-		BeforeInboxAcc:         [32]byte{1},
-		AfterInboxAcc:          [32]byte{2},
-		AfterDelayedAcc:        userDelayed.AfterInboxAcc(),
-		AfterDelayedCount:      2,
-		TimeBounds:             bridgegen.IBridgeTimeBounds{},
-		rawLog:                 types.Log{},
-		dataLocation:           0,
-		bridgeAddress:          [20]byte{},
-		serialized:             serializedUserMsgBatch,
-	}
-	emptyBatch := &SequencerInboxBatch{
-		BlockHash:              [32]byte{},
-		ParentChainBlockNumber: 0,
-		SequenceNumber:         2,
-		BeforeInboxAcc:         [32]byte{2},
-		AfterInboxAcc:          [32]byte{3},
-		AfterDelayedAcc:        userDelayed.AfterInboxAcc(),
-		AfterDelayedCount:      2,
-		TimeBounds:             bridgegen.IBridgeTimeBounds{},
-		rawLog:                 types.Log{},
-		dataLocation:           0,
-		bridgeAddress:          [20]byte{},
-		serialized:             serializedUserMsgBatch,
-	}
-	err = tracker.AddSequencerBatches(ctx, nil, []*SequencerInboxBatch{initMsgBatch, userMsgBatch, emptyBatch})
-	Require(t, err)
-
-	// Reorg out the user delayed message
-	err = tracker.ReorgDelayedTo(1, true)
```

Comment:
Instead of just removing this test, we should instead have this test insert an alternative delayed message at position 1 which would cause the same reorg. Instead of getting the delayed count below, we could then get the delayed message to ensure it changed.